WebKitBrowser/CMakeLists.txt

```diff
@@ -56,12 +28,16 @@ option(PLUGIN_WEBKITBROWSER_SEARCH_AND_DISCOVERY_APP "Include Search&Discovery A
 option(PLUGIN_WEBKITBROWSER_HTML_APP "Include HTML App in seperate plugin." OFF)
 option(PLUGIN_WEBKITBROWSER_LIGHTNING_APP "Include Lightning App in seperate plugin." OFF)
 
+option(PLUGIN_WEBKITBROWSER_ENABLE_JIT "Enable the use of JIT javascript optimalization" ON)
+option(PLUGIN_WEBKITBROWSER_ENABLE_DFG "Enable the use of DFG javascript optimalization" ON)
+
 set(PLUGIN_WEBKITBROWSER_AUTOSTART false CACHE STRING "Automatically start WebKitBrowser plugin")
 set(PLUGIN_WEBKITBROWSER_TRANSPARENT false CACHE STRING "Set transparency")
 set(PLUGIN_WEBKITBROWSER_STARTURL "about:blank" CACHE STRING "Initial URL for WebKitBrowser plugin")
-set(PLUGIN_WEBKITBROWSER_USERAGENT "" CACHE STRING "User agent string")
```

Comment:
Fine by me, to set another default here as long as we can override it from the config, I am fine.

Reply:
We might have a bit too fast with stating we can accept the empty default. This will break a lot of existing builds as they assume the default user agent as was set before (therefore introduced here again). 
We propose to set it how it always was and that you set it empty in your builds.

Reply:
> This will break a lot of existing builds as they assume the default user agent as was set before (therefore introduced here again).

How this will break existing builds? You'll anyways have to do a change build recipes to pull from rdkservices, so can you set the UA in build recipes as well? 

Just curious what is there a reason for defaulting to MacOS user agent?

Reply:
It will break builds as the repo where the source code is set from the makefiles and the user agent override would need to be set in the Buildroot defconfig file, so they are not stored in the same file. 
We looked at it and probably it would need so be set in most build files anyway is not such a huge problem to change it, so we'll set the default to empty.